_includes/_code/clickbox-a/main.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+// Uncomment lines 3 and 6 to view the visual layout at runtime.
+//import 'package:flutter/rendering.dart' show debugPaintSizeEnabled;
+
+void main() {
+  //debugPaintSizeEnabled = true;
+  runApp(new MyApp());
+}
+
+// ClickboxA manages its own state.
+class ClickboxA extends StatefulWidget {
+  ClickboxA({Key key}) : super(key: key);
+
+  _ClickboxAState createState() => new _ClickboxAState();
+}
+
+class _ClickboxAState extends State<ClickboxA> {
```

Comment:
Yeah, I'm worried about that, too. We're trying to find the line between "pretend this is in a very large app with Dependency Injection, private, abstractions, libraries, etc" and "this is how you respond to a button click", so we can focus on the important targeted concept for that specific part of the doc.

Maybe it's time to create a "style guide for snippets on the website". That would be pretty useful. I'll open an issue for that.